de.wagnermartin.Plattenalbum.json

```diff
@@ -0,0 +1,48 @@
+{
+  "app-id": "de.wagnermartin.Plattenalbum",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "command": "plattenalbum",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--share=network",
+    "--filesystem=xdg-music:ro",
+    "--filesystem=xdg-run/mpd",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.freedesktop.FileManager1",
+    "--own-name=org.mpris.MediaPlayer2.Plattenalbum"
+  ],
+  "modules": [
+    {
+      "name": "python3-mpd2",
+      "sources": [{
+        "type": "file",
+        "url": "https://files.pythonhosted.org/packages/53/be/e77206eb35eb37ccd3506fba237e1431431d04c482707730ce2a6802e95c/python-mpd2-3.1.1.tar.gz",
+        "sha256": "4baec3584cc43ed9948d5559079fafc2679b06b2ade273e909b3582654b2b3f5"
+      }],
+      "buildsystem": "simple",
+      "build-commands": [
+        "pip3 install --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"python-mpd2\""
+      ]
+    },
+    {
+      "name": "plattenalbum",
+      "sources": [{
+        "type": "git",
+        "url": "https://github.com/SoongNoonien/plattenalbum.git",
+        "tag": "v2.0.0",
+        "commit": "9cf9d8f5af471fbaa882888e0dab14d688a1b8da"
+      }],
+      "buildsystem": "simple",
+      "build-commands": [
+        "meson setup builddir --prefix=/app",
+        "ninja -C builddir install",
+        "glib-compile-schemas /app/share/glib-2.0/schemas"
```

Comment:
Cant this be done in up-stream?

https://mesonbuild.com/Gnome-module.html#gnomepost_install

Reply:
> Can't this be done in up-stream?

Done!